blogs/ecs-cross-account/account_backend/redis_and_database.yml

```diff
@@ -0,0 +1,331 @@
+Parameters:
+  ProjectName:
+    Type: String
+    Description: Project name to link stacks
+    Default: am-ecs-multi-account
+
+  EnvoyImage:
+    Type: String
+    Description: Envoy container image
+    Default: 840364872350.dkr.ecr.eu-central-1.amazonaws.com/aws-appmesh-envoy:v1.12.5.0-prod
+
+  RedisImage:
+    Type: String
+    Description: Container image for Redis
+    Default: redis:4.0.2
+
+  DatabaseImage:
+    Type: String
+    Description: Container image for Database
+    Default: mreferre/yelb-db:0.5
+
+  MeshName:
+    Type: String
+    Description: Name of the Mesh
+    Default: yelb
+
+Resources:
+  TaskSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: "Security group for the tasks"
+      VpcId:
+        Fn::ImportValue: !Sub "${ProjectName}:VPC"
+      SecurityGroupIngress:
+        - CidrIp:
+            Fn::ImportValue: !Sub "${ProjectName}:VpcCIDR"
+          IpProtocol: -1
+
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "${ProjectName}-log-group"
+      RetentionInDays: 30
```

Comment:
This is for a demo, make retention days less as the costs can add up. No more than a couple of days IMO

Reply:
Done, 5 days now.